portfolio/src/main/java/com/google/sps/servlets/TranslateAPIServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import java.util.HashSet;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslateAPIServlet extends HttpServlet{
+
+    HashSet<String> acceptedLanguages = new HashSet<String>();
```

Comment:
This should be private static.
In java consts are stored as static, since they don't need to change. Also private, since it does not have to be accessed from outside this class.
private static class level const vars are always UPPER_CASE. So
private static Set<String> ACCEPTED_LANGUAGES = ...

Reply:
Got it!